tools/OpenTelemetryYoutube/OpenTelemetryYoutube/Program.cs

```diff
@@ -16,16 +16,22 @@ class Program
     {
         // https://www.youtube.com/channel/UCHZDBZTIfdy94xMjMKz-_MA
         private const string openTelemetryChannelId = "UCHZDBZTIfdy94xMjMKz-_MA";
+        private const string governancePlaylistId = "PLVYDBkQ1Tdyzg1CuQgd9mdjwOUYg7ECYR";
```

Comment:
would be great to have this generalized. Technical committee meetings would be next